java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -201,4 +203,142 @@ public CompletableFuture<String> set(String key, String value, SetOptions option
+     * @param key - The key to decrement its value.
+     * @return the value of `key` after the decrement. An error is raised if `key` contains a value of
+     *     the wrong type or contains a string that can not be represented as integer.
+     */
+    @Override
+    public CompletableFuture<Long> decr(String key) {
+        Command command =
+                Command.builder()
+                        .requestType(Command.RequestType.DECR)
+                        .arguments(new String[] {key})
+                        .build();
+        return commandManager.submitNewCommand(command, BaseClient::handleLongResponse);
+    }
+
+    /**
+     * Decrements the number stored at `key` by `amount`. If `key` does not exist, it is set to 0
+     * before performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/decrby/">redis.io</a> for details.
+     * @param key - The key to decrement its value.
+     * @param amount - The amount to decrement.
+     * @return the value of `key` after the decrement. An error is raised if `key` contains a value of
+     *     the wrong type or contains a string that can not be represented as integer.
+     */
+    @Override
+    public CompletableFuture<Long> decrBy(String key, long amount) {
+        Command command =
+                Command.builder()
+                        .requestType(Command.RequestType.DECR_BY)
+                        .arguments(new String[] {key, Long.toString(amount)})
+                        .build();
+        return commandManager.submitNewCommand(command, BaseClient::handleLongResponse);
+    }
+
+    /**
+     * Increments the number stored at `key` by one. If `key` does not exist, it is set to 0 before
+     * performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/incr/">redis.io</a> for details.
+     * @param key - The key to increment its value.
+     * @return the value of `key` after the increment, An error is raised if `key` contains a value of
+     *     the wrong type or contains a string that can not be represented as integer.
+     */
+    @Override
+    public CompletableFuture<Long> incr(String key) {
+        Command command =
+                Command.builder()
+                        .requestType(Command.RequestType.INCR)
+                        .arguments(new String[] {key})
+                        .build();
+        return commandManager.submitNewCommand(command, BaseClient::handleLongResponse);
+    }
+
+    /**
+     * Increments the number stored at `key` by `amount`. If `key` does not exist, it is set to 0
+     * before performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/incrby/">redis.io</a> for details.
+     * @param key - The key to increment its value.
+     * @param amount - The amount to increment.
+     * @returns the value of `key` after the increment, An error is raised if `key` contains a value
```

Comment:
Important: the tag should be `@return` - please update everywhere
```suggestion
     * @return The value of `key` after the increment, An error is raised if `key` contains a value
```